api/v1alpha2/awscluster_types.go

```diff
@@ -41,6 +41,17 @@ type AWSClusterSpec struct {
 	// ones added by default.
 	// +optional
 	AdditionalTags Tags `json:"additionalTags,omitempty"`
+
+	// ControlPlaneSpec is optional configuration for customizing control plane behavior
```

Comment:
Is this supposed to be ControlPlaneSpec with a nested LoadBalancerSpec, or just ControlPlaneLoadBalancer?